parts/kubernetesmastercustomscript.sh

```diff
@@ -130,6 +130,23 @@ function ensureKubelet() {
 function extractKubectl(){
     systemctl enable kubectl-extract
     systemctl restart kubectl-extract
+    kubectlExtracted=1
```

Comment:
This fix still feels brittle. It doesn't feel right to create a systemd service to be scheduled by systemd, and then have this manual loop to keep kicking it until it does the right thing.

Can we just remove the systemd unit? Or get systemd to restart it and drive it to success instead of us having to do it manually?